pom.xml

```diff
@@ -45,7 +45,7 @@
     <scala.version>2.12.15</scala.version>
     <spark.major.version>3</spark.major.version>
     <sparkbundle.version>3.2</sparkbundle.version>
-    <spark.version>3.4.2</spark.version>
+    <spark.version>3.5.1</spark.version>
```

Comment:
it's a bit hurry to change the default Spark version, I think leave it to 3.4.2 is better for now

Reply:
@ulysses-you, do we need this line? I note each profile already specifies its spark version, including the default activated profile, spark-3.2.

Reply:
It seems we can delete this line.

Reply:
it can make idea happy without profile..

Reply:
Revert to 3.4.2.